src/text.ts

```diff
@@ -151,11 +151,11 @@ export const login = async ({ phone, code, logger, ip }: ILogin): Promise<string
 
     // TODO
     // if (yamlConfig.carrierRegexFilter)  {
-    // 
+    //
     // }
     //
     // only fetch info once
-    if (user.twilio.countryCode == undefined) {
+    if (user.twilio.countryCode === undefined) {
```

Comment:
may worth check this is not null returned

Reply:
We can add a null check here to be safe.